docker_install.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+
+apt-get update && apt-get install -y apt-transport-https ca-certificates curl software-properties-common
+curl -fsSL https://mirrors.aliyun.com/docker-ce/linux/ubuntu/gpg | apt-key add -
+add-apt-repository 'deb [arch=amd64] https://mirrors.ustc.edu.cn/docker-ce/linux/ubuntu bionic stable'
+apt-get update && apt-get install -y docker-ce
+
+mkdir -p /etc/docker
+echo "{\n\t\"runtimes\": {\n\t\t\"rune\": {\n\t\t\t\"path\": \"/usr/local/bin/rune\",\n\t\t\t\"runtimeArgs\": []\n\t\t}\n\t},\n\t\"storage-driver\": \"vfs\"\n}" >> /etc/docker/daemon.json
```

Comment:
There is no need to use `rune`. Since we use enclave-cc project to deploy cvtpm, enclave-cc use `runc` runtime rather than `rune`.